lessons/module-2/context-and-this-keyword.md

```diff
@@ -151,32 +151,6 @@ Taking turns for each prompt in driver/navigator fashion, use the code snippet b
 ```
 </section>
 
-### Are there other rules?
-
-Here's one more scenario to look at
-
-<section class="call-to-action">
-### In Your Console
-
-Run the following command!
-
-```js
-function logThis() {
-  console.log(this);
-}
-
-logThis();
-```
-
-* What is the value of this?  What is one final rule we can add that applys to *this*?
-</section>
-
-<section class="answer">
-### Rule #3  
-
-By default, *this* refers to the global object (or in the browser, the window).
-</section>
-
```

Comment:
Good call removing this section since it's covered in the first example now.  Looks great!